src/cli/download_tracker.rs

```diff
@@ -111,13 +111,13 @@ impl DownloadTracker {
         self.total_downloaded += len;
         self.downloaded_this_sec += len;
 
-        let current_time = precise_time_s();
+        let current_time = Instant::now();
 
         match self.last_sec {
             None => self.last_sec = Some(current_time),
```

Comment:
This last_sec seems to do nothing except to save the last current time.
Should we remove it?